### PR TITLE
test

### PR DESCRIPTION
**Pull request by ehuss**
r? @grashgal

### CODE REVIEW

**Comment by ehuss** [comment]
Failed to set assignee to `grashgal`: invalid assignee

**Review by ehuss — COMMENTED** [review]
r? @grashgal

**Comment by ehuss** [comment]
Failed to set assignee to `grashgal`: invalid assignee

File: bar/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Testing
```
Comment (ehuss) [inline bar/README.md:1]: r? @grashgal

**Comment by ehuss** [comment]
Failed to set assignee to `grashgal`: invalid assignee

File: bar/README.md (hunk 1 above)
Comment (ehuss) [reply bar/README.md:1]: r? @grashgal

**Comment by ehuss** [comment]
Failed to set assignee to `grashgal`: invalid assignee

File: bar/README.md (hunk 1 above)
Comment (ehuss) [reply bar/README.md:1]: r? @grashgal

**Comment by ehuss** [comment]
Failed to set assignee to `grashgal`: invalid assignee

> **Note**: Only org members, users with write permissions, or people who have commented on the PR may be assigned.

**Comment by ehuss** [comment]
r? rust-lang/cargo

**Comment by ehuss** [comment]
r? ehuss/group2

**Comment by ehuss** [comment]
Failed to set assignee to `octocat`: invalid assignee

> **Note**: Only org members, users with write permissions, or people who have commented on the PR may be assigned.

**Comment by ehuss** [comment]
r? group2

**Comment by ehuss** [comment]
r? group2

**Comment by ehuss** [comment]
Failed to set assignee to `octocat`: invalid assignee

> **Note**: Only org members, users with write permissions, or people who have commented on the PR may be assigned.

**Comment by ehuss** [comment]
Failed to set assignee to `EHUSS`: invalid assignee

> **Note**: Only org members, users with write permissions, or people who have commented on the PR may be assigned.

**Comment by ehuss** [comment]
Failed to set assignee to `EHUSS`: invalid assignee

> **Note**: Only org members, users with write permissions, or people who have commented on the PR may be assigned.

**Comment by grashgal** [comment]
r? EHUSS

**Comment by grashgal** [comment]
r? eHuss

**Comment by ehuss** [comment]
Failed to set assignee to `OCTOCAT`: invalid assignee

> **Note**: Only org members, users with write permissions, or people who have commented on the PR may be assigned.

**Comment by grashgal** [comment]
r? ghost

**Comment by ehuss** [comment]
Failed to set assignee to `ghost`: invalid assignee

> **Note**: Only org members, users with write permissions, or people who have commented on the PR may be assigned.

**Comment by ehuss** [comment]
r? group3

**Comment by grashgal** [comment]
comment

**Comment by ehuss** [comment]
r? grashgal

**Comment by ehuss** [comment]
Failed to set assignee to `ghost`: invalid assignee

> **Note**: Only org members, users with write permissions, or people who have commented on the PR may be assigned.

**Comment by ehuss** [comment]
r? ehuss/somegroup

**Comment by ehuss** [comment]
r? ehuss/somegroup

**Comment by ehuss** [comment]
Team or group `ehuss/somegroup` not found.

**Comment by ehuss** [comment]
r? rust-lang/team-does-not-exist

**Comment by ehuss** [comment]
Team or group `rust-lang/team-does-not-exist` not found.

**Comment by ehuss** [comment]
Failed to set assignee to `fjsdklajteoiwajgdlnasnv`: invalid assignee

> **Note**: Only org members, users with write permissions, or people who have commented on the PR may be assigned.

**Pull request closed without merging**